MockKitUI/View Controllers/MockCGMManagerSettingsViewController.swift

```diff
@@ -15,13 +16,28 @@ import MockKit
 final class MockCGMManagerSettingsViewController: UITableViewController {
     let cgmManager: MockCGMManager
 
-    private var glucoseUnit: HKUnit
+    private var displayGlucoseUnitObservable: DisplayGlucoseUnitObservable
 
-    init(cgmManager: MockCGMManager, glucoseUnit: HKUnit) {
+    private var cancellable: AnyCancellable?
```

Comment:
[nit] I prefer using `private lazy var cancellables = Set<AnyCancellable>()` and `.store(in: &cancellables)`, but that's just me.